<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UartToolClass</class>
 <widget class="QMainWindow" name="UartToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UartTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_portName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baudRate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Uart</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comopen">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textEdit_read">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>361</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_write">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1E 01 00 00 00 00 00 01 00 00 0F 2E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OpenBin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearBin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UartTool.qrc"/>
 </resources>
 <connections/>
</ui>
